packages/metadata/README.md

```diff
@@ -0,0 +1,35 @@
+# @loopback/metadata
+
+This module contains utilities to manipulate TypeScript metadata. It includes:
```

Comment:
It would be nice to include a short explanation of the common mistakes people make when using reflect-metadata and how this package is taking care of those tricky parts.